docs/api.yaml

```diff
@@ -0,0 +1,1061 @@
+openapi: "3.0.3"
+info:
+  #  description:
+
+  version: "1.0.0"
+  title: "HEI Admin API"
+  contact:
+    email: "contact@hei.school"
+
+  #servers:
+  # - description: To complete
+  #   url: example.com
+
+paths:
+  /groups:
+    get:
+      tags:
+        - "Groups"
+      summary: "Find groups based on search criteria"
+      parameters:
+        - name: "id"
+          in: "query"
+          description: "Identifier of the group to find"
+          required: false
+          schema:
+            type: string
+        - name: offset
+          in: query
+          required: false
+          description: "Zero-based offset for pagination. Defaults to `0`."
+          schema:
+            type: integer
+        - name: limit
+          in: query
+          required: false
+          description: "Pagination limit, that is the maximum number of elements that can be retrieved. Defaults to `1000` and cannot exceed `1000`."
+          schema:
+            type: integer
+
+      operationId: "findGroups"
+      responses:
+        200:
+          description: "List of groups"
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Group'
+        400:
+          $ref: "#/components/responses/400"
+        403:
+          $ref: "#/components/responses/403"
+        404:
+          $ref: "#/components/responses/404"
+        429:
+          $ref: "#/components/responses/429"
+        500:
+          $ref: "#/components/responses/500"
+
+    post:
+      tags:
+        - "Groups"
+      summary: "Create new groups"
+      description: ""
+      operationId: "createGroups"
+      requestBody:
+        description: "The collection of groups to create"
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateGroup'
+      responses:
+        200:
+          description: "Collection of created groups"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Group'
+        400:
+          $ref: "#/components/responses/400"
+        403:
+          $ref: "#/components/responses/403"
+        404:
+          $ref: "#/components/responses/404"
+        429:
+          $ref: "#/components/responses/429"
+        500:
+          $ref: "#/components/responses/500"
+
+    patch:
+      tags:
+        - "Groups"
+      summary: "Update an existing group"
+      parameters:
```

Comment:
Add the creation_date parameter

Reply:
I actually removed creation_date attribute according to the past conversation about the from/to parameter not needed yet, for ALL schemas.

Can you explain what is it use for ?

Reply:
creation_date is useful for identifying to which "promo" a group belongs to. We just don't need a search filter on that attribute since there are not that many groups. It's good that the search filter was removed, but keep the information nethertheless.